platforms/core-configuration/configuration-cache/src/integTest/groovy/org/gradle/configurationcache/isolated/IsolatedProjectsAccessFromGroovyDslIntegrationTest.groovy

```diff
@@ -706,7 +706,6 @@ class IsolatedProjectsAccessFromGroovyDslIntegrationTest extends AbstractIsolate
         "resources.text.fromFile('1.txt', 'UTF-8')"                | ""
         "fromTask"                                                 | "def fromTask = new Object() { def buildDependencies = tasks.help.taskDependencies }"
         "artifacts.add('default', new File('a.txt'))"              | "configurations.create('default')"
-        "dependencies.project([path: ':', configuration: 'test'])" | "plugins { id('java') }"
```

Comment:
Why was this removed?

Reply:
This causes a deprecation warning and it appears I was too lazy to add the `expectDocumentedDeprecationWarning` call